### PR TITLE
Don't create a superuser if already exists

### PR DESCRIPTION
**Pull request by bogdal**
This PR allows to run `./manage.py populatedb --createsuperuser` several times in a row.

### CODE REVIEW

**Comment by patrys** [comment]
You can't catch integrity errors without creating a transaction save point and doing a rollback as otherwise you invalidate the entire connection. See how `get_or_create` works.

**Comment by bogdal** [comment]
@patrys Please review and merge.

File: saleor/product/management/commands/populatedb.py [hunk 1]
```diff
@@ -26,6 +26,15 @@ def handle(self, *args, **options):
             self.stdout.write(msg)
 
         if options['createsuperuser']:
-            user = User.objects.create_superuser(
-                email='admin@example.com', password='admin')
-            self.stdout.write('Superuser - %s' % user.email)
+            credentials = {'email': 'admin@example.com', 'password': 'admin'}
+            user, created = User.objects.get_or_create(
+                email=credentials['email'],
+                is_active=True, is_staff=True, is_superuser=True)
```
Comment (patrys) [inline saleor/product/management/commands/populatedb.py]: Shouldn't these be moved to `defaults`? What if the user with that email exists but is not active or is not staff?

Comment (bogdal) [inline saleor/product/management/commands/populatedb.py]: Good point :+1:

**Pull request merged**